.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,249 @@
+      working-directory: neofs-dev-env
+
+    # Generate hosts
+    - name: Generate hosts
+      run: |
+        sudo ./bin/update_hosts.sh
+      working-directory: neofs-dev-env
+
+    # Prepare test environment
+    - name: Prepare test environment
+      run: |
+        make prepare-test-env
+      working-directory: neofs-dev-env
+
+###############################################################################
+###############################################################################
+###############################################################################
+    # Make venv
+    - name: Prepare venv
+      run: |
+        make venv.local-pytest
+      working-directory: neofs-testcases
+
+    # Run tests
+    - name: Run tests
+      run: |
+        source venv.local-pytest/bin/activate && pytest --alluredir=allure-results pytest_tests/testsuites
+      continue-on-error: true
+      working-directory: neofs-testcases
+
+###############################################################################
+###############################################################################
+###############################################################################
+    # Cleanup tests
+    - name: Cleanup test environment
+      if: always()
+      run: |
+        make clean
+      continue-on-error: true
+      working-directory: neofs-testcases
+
+    # Cleanup test environment
+    - name: Cleanup test environment
+      if: always()
+      run: |
+        make down
+        make clean
+      continue-on-error: true
+      working-directory: neofs-dev-env
+
+    # Cleanup directories before finish
+    - name: Cleanup directories before finish
+      if: always()
+      run: |
+        echo "Clean"
+        rm -rf ${{ env.CONTRACT_DIR }}
+        rm -rf ${{ env.LOCODE_DIR }}
+      continue-on-error: true
+
+###############################################################################
+###############################################################################
+###############################################################################
+    - name: Get Allure history
+      uses: actions/checkout@v2
+      if: always()
+      continue-on-error: true
+      with:
+        ref: gh-pages
+        path: gh-pages
+
+    - name: Allure Report action from marketplace
+      uses: simple-elf/allure-report-action@master
+      if: always()
+      id: allure-report
+      with:
+        allure_results: allure-results
+        allure_report: allure-report
+        gh_pages: gh-pages
+        allure_history: allure-history
+
+    - name: Deploy report to Github Pages
+      if: always()
+      uses: peaceiris/actions-gh-pages@v2
+      env:
+        PERSONAL_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PUBLISH_BRANCH: gh-pages
+        PUBLISH_DIR: allure-history
+
+    - name: Post the link to the report
+      if: always()
+      uses: Sibz/github-status-action@v1
+      with:
+        authToken: ${{secrets.GITHUB_TOKEN}}
```

Comment:
The same token as for GH Pages report? Have you tasted it?

Reply:
Hm, I've checked, there's no such secret in the neofs-node repo, we need to create it and test publication with this secret.

Reply:
> Hm, I've checked, there's no such secret in the neofs-node repo, we need to create it and test publication with this secret.

GitHub automatically creates a GITHUB_TOKEN secret

Reply:
> The same token as for GH Pages report? Have you tasted it?

Yes, I tested it: https://github.com/vvarg229/neofs-node/actions/workflows/run-tests.yml